contribs/github-bot/internal/client/client.go

```diff
@@ -0,0 +1,248 @@
+package client
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/gnolang/gno/contribs/github-bot/internal/logger"
+	p "github.com/gnolang/gno/contribs/github-bot/internal/params"
+
+	"github.com/google/go-github/v64/github"
+)
+
+// PageSize is the number of items to load for each iteration when fetching a list
+const PageSize = 100
+
+var ErrBotCommentNotFound = errors.New("bot comment not found")
+
+type GitHub struct {
```

Comment:
Can you put in some godoc? From what I understand, we should use this type to put in methods to access the GitHub API, which are more complex than single calls to it.

Reply:
Done 👍